shared-doc/build-and-run-the-quickstart-with-bootable-jar.adoc

```diff
@@ -34,6 +35,51 @@ The quickstart `pom.xml` file contains a Maven profile named *bootable-jar* whic
           </build>
       </profile>
 ----
+endif::portedToGlow[]
+ifdef::portedToGlow[]
+[source,xml,subs="attributes+"]
+----
+      <profile>
+          <id>bootable-jar</id>
+          <build>
+              <plugins>
+                  <plugin>
+                      <groupId>org.wildfly.plugins</groupId>
+                      <artifactId>wildfly-jar-maven-plugin</artifactId>
```

Comment:
should be wildfly-maven-plugin

Reply:
Perhaps we need the version too 5.0.0.Beta2, the latest one is 4.2.x that doesn't support glow.